frog/imports/ui/GraphEditor/RemoteControllers/RemoteLibrary.jsx

```diff
@@ -0,0 +1,142 @@
+// @flow
+import React, { Component } from 'react';
+
+import { importGraph, collectGraphs } from '/imports/api/remoteGraphs';
+import { collectActivities, importAct } from '/imports/api/remoteActivities';
+import LibraryListComponent from './LibraryListComponent';
+
+const filterWithStr = (list: Array<any>, searchStr: string) =>
+  list &&
+  list
+    .filter(
+      x =>
+        (x.activity_type &&
+          x.activity_type.toLowerCase().includes(searchStr.toLowerCase())) ||
```

Comment:
you could do const lowSearch = searchStr.toLowerCase() above, to avoid repetition